<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>create_client</class>
 <widget class="QWidget" name="create_client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>250</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Criar Cliente</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>448</width>
     <height>169</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Nome do Cliente:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line_name"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Numero:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line_numero"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>NIF:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line_nif"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>ENDERECO:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line_adress"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Inicio do Contrato:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="date_start"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>ID DA MOTO:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_id_moto"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
